cloudflare_worker/eventlog.md

```diff
@@ -9,3 +9,4 @@
 | 2018-09-26  | Jeff LaJoie  | Priorities for force > query > random/crawler    |
 | 2018-09-26  | Jeff LaJoie  | Enables random weight, updates random string     |
 | 2018-19-24  | Jeff LaJoie  | Moves to 100% rollout, sets crawlers to TEST     |
+| 2018-19-24  | Jeff LaJoie  | Removes querystring override, forces rollout     |
```

Comment:
This is not a date.  Neither is the one above :)

Reply:
https://www.youtube.com/watch?v=tQgg6fpV_Kg